### DISCUSSION TITLE
Orphaned Recipes

### DISCUSSION THREAD

**Discussion by Gilimi-the-Grim** [mealie-recipes/mealie#1792] [Feature Request]
Hello all,

First I want to say that I am new to Mealie, but what I have used so far has been great.  My wife likes that her recipes are available on the all in one she sits on the counter.

In "playing" with Mealie I noticed that a recipe can get orphaned.  For example, say I create a user and that user creates a recipe.  If you delete the user the recipe becomes orphaned if there is no one in that group.

As a feature, it would be nice if the site administrator can move recipes to a different group, assign a recipe to a new owner and / or delete orphaned recipes.

I am trying to use Mealie in the following manner.

Create a group for each family unit, where they can store their recipes.  The recipes should be private to the group unless it is marked public.  But marking the recipes public has no effect that I can tell in the system.  I know Mealie is Beta and there is much to be completed.  So rather than muddy the water with several requests, I will wait and see how the development goes.

Thank you to all the developers on this project.  I find it very useful so far and hope to be able to contribute in some small manner someday.

Regards,
Joe

**Comment by michael-genson** [comment]
Related to this: https://github.com/mealie-recipes/mealie/issues/3219

I added a note to explicitly cover this usecase
